packages/kbn-ftr-common-functional-services/services/saml_auth/saml_auth_provider.ts

```diff
@@ -148,7 +148,7 @@ export function SamlAuthProvider({ getService }: FtrProviderContext) {
         if (role === CUSTOM_ROLE && !isCustomRoleEnabled) {
           throw new Error(`Custom roles are not supported for the current deployment`);
         }
-        const roleDescriptor = supportedRoleDescriptors.get(role);
+        const roleDescriptor = { ...supportedRoleDescriptors.get(role), metadata: {} };
```

Comment:
In retrospect, we can just set metadata to undefined.  @dmlemeshko should we do this here, or in the createApiKeyPayload function?

```suggestion
        const roleDescriptor = { ...supportedRoleDescriptors.get(role), metadata: undefined };
```

Reply:
Disregsard. I made an update in 8bec6930dfba4ba31c0d52329553023ac3dca540 because I failed to account for the undefined check. This should now resolve some of the additional FTR failures we're seeing.